i18n/locales/en.ts

```diff
@@ -2737,49 +2737,52 @@ Stack Hint: To spend before the secret is revealed, Vanderpoole uses his signatu
     },
     opening_a_channel_four: {
       title: 'Multisig',
       nav_title: 'Multisig',
       heading_one: 'Multisig',
-      paragraph_one: `This game will have to start with a 2-of-2 multisig, confirmed on the blockchain. Once we have that we can figure out a way to make that single on-chain transaction work harder. To do more with less.`,
+      paragraph_one: `Before paying Laszlo, let's first get the funds into a 2-of-2 multisig, confirmed on the blockchain. Then we can figure out a way to make that single on-chain transaction work harder—to do more with less.`,
+      heading_two: 'Instructions',
       multisig_one: 'Fill in the output amount and the output script',
       multisig_two: 'Sign the input by clicking "Sign"',
-      heading_two: 'Hints',
+      heading_three: 'Hints',
       hint_one_a: `You and Laszlo want to work together in the future to spend output 0 with <br/> `,
       hint_one_b: `<span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base"> 0 SIG(LASZLO) SIG(YOU) </span>`,
+      hint_two: `Don't forget to set aside miner fees`,
     },
     opening_a_channel_five: {
-      title: 'Refund Protection in Multisig',
-      nav_title: 'Refund Protection in Multisig',
-      paragraph_one: `—LASZLO:  "OK, thanks, this is a good start. But what if I walk away from the table and you never see me again? I could broadcast this transaction, and your 100,000 satoshis would be stuck in a 2-key multisig from which you could never recover.`,
+      title: 'Refund protection in multisig',
+      nav_title: 'Refund protection in multisig',
+      paragraph_one: `—LASZLO:  "OK, thanks. This is a good start. But what if I walk away from the table and you never see me again? I could broadcast this transaction, and your 100,000 satoshis would be stuck in a 2-key multisig from which you could never recover.`,
       paragraph_two: `Do yourself a favor; before signing this, make a refund transaction to know you can recover your money."`,
     },
     updating_the_state_one: {
       title: 'The refund',
       nav_title: 'The refund',
       heading_one: 'The refund',
-      paragraph_one: `A new tab appears: <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">Refund</span> which starts another TX template on the same screen, with an arrow from the <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">Multi-Sig</span> output to this TX input.`,
+      paragraph_one: `A new tab appears on your ePhone Infinity: <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">Refund</span> which starts another TX template on the same screen, with an arrow from the <span className="rounded-sm px-1.5 py-1 h-[28px] font-mono bg-[#0000004D] m-0.5 text-base">Multisig</span> output to this TX input.`,
```

Comment:
@pinheadmz Here's where I updated it to mention your ePhone Infinity :)

Reply:
This is great -- I think it would help to remind the user to deduce another 1,000 sat fee. The validation code does check for it (sats==99000 for correct answer)

Reply:
I addressed this by updating the help page to say

> Don't forget to set aside 1,000 sats from your change for miner fees.